.github/workflows/test.yml

```diff
@@ -55,7 +64,7 @@ jobs:
       - name: upload coverage
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: php vendor/bin/php-coveralls --coverage_clover=build/logs/clover.xml -v
+        run: php vendor/bin/php-coveralls --exclude-no-stmt --coverage_clover=build/logs/clover.xml -v
```

Comment:
I could not find any information about the option you added. I suppose this is fine but could you explain what it does?

Reply:
Without this option, interfaces were included and generated a single uncovered line.
Documentation is available at https://github.com/php-coveralls/php-coveralls#cli-options

Reply:
I seem to have overlooked that in the docs.